src/App.tsx

```diff
@@ -1,55 +1,53 @@
 import "./App.css";
-import { useState, useEffect } from "react";
-import { Card, Person } from "./components/Card/Card";
-import { Wrapper } from "./components/Wrapper/Wrapper";
-import { Menu } from "./components/Menu/Menu";
-import { Form } from "./components/Form/Form";
-// import { Person, cardsData } from "./components/Card/data/cards-data";
-import { useAPI } from "./components/test/useApi";
-import { TestButton } from "./components/TestButton";
-import { Cart } from "./components/Cart/Cart";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 
+import { MainLayout } from "./layouts/MainLayout";
+import { HomePage } from "./pages/HomePage";
+import { ClientsPage } from "./pages/ClientsPage";
+import { ClientPage } from "./pages/ClientPage";
+import { AddClientPage } from "./pages/AddClientPage";
+import { EditClientPage } from "./pages/EditClientPage";
+import { OrdersPage } from "./pages/OrdersPage";
+import { InvoicesPage } from "./pages/InvoicesPage";
+import { PostsPage } from "./pages/PostsPage";
+import { NotFoundPage } from "./pages/NotFoundPage";
+import { ROUTES } from "./routes";
 
+import { QueryClient, QueryClientProvider, QueryCache } from "@tanstack/react-query";
+import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
 
-function App() {
-  const { data,isLoading,isError } = useAPI<Person[]>("https://jsonplaceholder.typicode.com/users");
-  console.log(data);
-  // const ref=useRef<ElementRef<HTMLDivElement>>(null)
-
-  // const [filteredCardsData, setFilteredCardsData] = useState<Person[]>(cardsData);
-  // const handleFilterChange = (filteredData: Person[]) => {
-  //   setFilteredCardsData(filteredData);
-  // };
-  const [filteredCardsData, setFilteredCardsData] = useState<Person[]>([]);
-  useEffect(() => {
-    if (data) {
-      setFilteredCardsData(data);
-    }
-  }, [data]);
-
-  const handleFilterChange = (filteredData: Person[]) => {
-    setFilteredCardsData(filteredData);
-  };
-  if(isError){
-    return <p>Error</p>
-  }
-  if(isLoading){
-    return <p>Loading...</p>
-  }
+const queryClient = new QueryClient({
+  queryCache: new QueryCache(),
+  defaultOptions: {
+    queries: {
+      staleTime: Infinity,
```

Comment:
no te Infinity to jest czasem ryzykowne bo to trzyma tak długo aż użytkownik nie wyczyści przeglądarki